components/common/shoppingCart/utils.ts

```diff
@@ -20,7 +21,7 @@ const getTokenDecimal = (item: ShoppingCartItem) => {
   return chainPropListOf(token.toLowerCase()).tokenDecimals
 }
 
-export const totalPriceUsd = (items: ShoppingCartItem[]) => {
+export const totalPriceUsd = (items: ShoppingCartItem[] | ListCartItem[]) => {
```

Comment:
Then we can remove the shoppingCartItem[] since that has more fields

Reply:
true, the function will accept anything that extends the basic interface required for it to work

Reply:
Yeah but i think it will throw an error on the royalties since i don't have that. But i don't use it so it was okay for me.
Please advice how you'd want me to change this

Reply:
I can separate it in to two functions if you want

Reply:
First option with a type guard
<img width="789" alt="image" src="https://github.com/kodadot/nft-gallery/assets/3810177/1a68b541-e028-4d29-aa9a-39c01262929b">

Option 2 which i believe is redundant
<img width="750" alt="image" src="https://github.com/kodadot/nft-gallery/assets/3810177/c27c8311-95c5-4f25-aafc-2f0436aeac3a">

Reply:
Actually I forgot that i stopped using it. I'll revert it.